wpeverest/everest-forms
# Enhanced - Improve entries with different actions

### All Submissions:






### Changes proposed in this Pull Request:



Improve the entries actions and view for PRO version.

### How to test the changes in this Pull Request:

1. Surf entries and its single view.
2. You should see unread and starred related actions.
3. In a single view, you can see meta actions. That's all you need to test.

### Types of changes:




### Other information:




### Changelog entry

> Enhanced - Improve entries with different actions.
